pkgs/by-name/am/amdenc/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  autoPatchelfHook,
+}:
+
+let
+  amdgpuVersion = "6.1.3";
+  ubuntuVersion = "22.04";
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "amdenc";
+  version = "1.0-1787253";
+
+  src = fetchurl {
+    url = "https://repo.radeon.com/amdgpu/${amdgpuVersion}/ubuntu/pool/proprietary/liba/libamdenc-amdgpu-pro/libamdenc-amdgpu-pro_${finalAttrs.version}.${ubuntuVersion}_amd64.deb";
+    hash = "sha256-RSkWQ3g++uKVrk5J9R8WA6qL0f+B2z8/mlflQ/cQZcg=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    autoPatchelfHook
+  ];
+
+  buildInputs = [ stdenv.cc.cc ];
```

Comment:
What do we need a C compiler for here and why does it need to be built for the hostPlatform?

Reply:
Should have been `stdenv.cc.cc.lib` as it needs `libstdc++` and `libgcc_s`